frontend/src/pages/execution-detail.module.css

```
@@ -0,0 +1,106 @@
+.page {
+}
```

Stylelint (17.14.0)

[error] Empty block (block-no-empty)

(block-no-empty)